deepstream/vot/config.pbtxt.tf

```diff
@@ -8,25 +8,25 @@ input [
     name: "image_tensor"
     data_type: TYPE_UINT8
     format: FORMAT_NHWC
-    dims: [ 300, 300, 3 ]
+    dims: [ 256,256, 3 ]
   }
 ]
 output [
   {
     name: "detection_boxes"
     data_type: TYPE_FP32
-    dims: [ 40, 4]
-    reshape { shape: [40, 4] }
+    dims: [ 100, 4]
```

Comment:
which model was this?

Reply:
this is the VOT model we downloaded from Cloud console